src/link/MachO.zig

```diff
@@ -3958,6 +3961,12 @@ pub fn getDeclVAddr(self: *MachO, decl: *const Module.Decl) u64 {
     return self.locals.items[decl.link.macho.local_sym_index].n_value;
 }
 
+pub fn getGot(self: *MachO, ally: Allocator) !link.File.GotWithAddr {
+    _ = self;
+    _ = ally;
+    @panic("TODO GET GOT for MACOS");
```

Comment:
@kubkon I would appreciate help on how to get the got on macos to allow emit_asm to work on macos also.